cdap-docs/reference-manual/source/release-notes.rst

```diff
@@ -23,6 +23,140 @@ Cask Data Application Platform Release Notes
    :backlinks: none
    :depth: 2
 
+
+`Release 3.4.0 <http://docs.cask.co/cdap/3.4.0/index.html>`__
+=============================================================
+
+New Features
+------------
+- `CDAP-9999 <https://issues.cask.co/browse/CDAP-9999>`__ - Added...
+
+
+Improvements
+------------
+- `CDAP-9999 <https://issues.cask.co/browse/CDAP-9999>`__ - Added...
+
+
+Bug Fixes
+---------
+- `CDAP-9999 <https://issues.cask.co/browse/CDAP-9999>`__ - Fixed a problem with...
+
+
+API Changes
+-----------
+- `CDAP-9999 <https://issues.cask.co/browse/CDAP-9999>`__ - Added...
+
+
+Deprecated and Removed Features
+-------------------------------
+- See the :ref:`CDAP 3.4.0 Javadocs <javadocs>` for a list of deprecated and removed APIs.
+- `CDAP-9999 <https://issues.cask.co/browse/CDAP-9999>`__ - Added...
+
+
+.. _known-issues-340:
+
+Known Issues
+------------
+
+- After upgrading CDAP from a pre-3.0 version, any unprocessed metrics data in Kafka will
+  be lost and *WARN* log messages will be logged that tell about the inability to process
+  old data in the old format.
+
+- `CDAP-797 <https://issues.cask.co/browse/CDAP-797>`__ - 
+  When running secure Hadoop clusters, debug logs from MapReduce programs are not
+  available.
+
+- `CDAP-1007 <https://issues.cask.co/browse/CDAP-1007>`__ -
+  If the Hive Metastore is restarted while the CDAP Explore Service is running, the
+  Explore Service remains alive, but becomes unusable. To correct, restart the CDAP Master
+  |---| which will restart all services |---| as described under "Starting CDAP Services"
+  for your particular Hadoop distribution in the :ref:`Installation documentation <installation-index>`.
+
+- `CDAP-1587 <https://issues.cask.co/browse/CDAP-1587>`__ - 
+  CDAP internally creates tables in the "user" space that begin with the word
+  ``"system"``. User datasets with names starting with ``"system"`` can conflict if they
+  were to match one of those names. To avoid this, do not start any datasets with the word
+  ``"system"``. 
+
+- `CDAP-2632 <https://issues.cask.co/browse/CDAP-2632>`__ -
+  The application in the `cdap-kafka-ingest-guide 
+  <https://github.com/cdap-guides/cdap-kafka-ingest-guide/tree/release/cdap-3.0-compatible>`__ 
+  does not run on Ubuntu 14.x as of CDAP 3.0.x.
+
+- `CDAP-2721 <https://issues.cask.co/browse/CDAP-2721>`__ -
+  Metrics for :ref:`TimePartitionedFileSets <datasets-timepartitioned-fileset>` can show 
```

Comment:
That's just a symptom. Known issue is different. https://issues.cask.co/browse/CDAP-2721